semantic_conventions/trace/tls.yaml

```diff
@@ -0,0 +1,204 @@
+groups:
+  - id: tls
+    prefix: tls
+    type: span
+    brief: "This document defines semantic conventions for TLS/SSL client and server Spans."
+    attributes:
+      - id: version
+        requirement_level: required
+        brief: >
+          Numeric part of the version parsed from the original string of the
+          negotiated [SSL/TLS protocol version](https://www.openssl.org/docs/man1.1.1/man3/SSL_get_version.html#RETURN-VALUES)
+        type: string
+        examples: ["1.2", "3"]
+      - id: version_protocol
+        requirement_level: required
+        brief: >
+          Normalized lowercase protocol name parsed from original string of the
+          negotiated [SSL/TLS protocol version](https://www.openssl.org/docs/man1.1.1/man3/SSL_get_version.html#RETURN-VALUES)
+        type:
+          allow_custom_values: false
```

Comment:
any reason for this limitation? there's DTLS or there could be some other protocols in the future, so suggestion here is to allow custom values

Reply:
yes, I think that can be dropped. my initial assumption was that there is only ssl & tls, I have not thought about dtls or any other option